lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -166,6 +168,21 @@ def create(self, subsection, course_structure, course):
             self._compute_and_save_grade(subsection, course_structure, course)
         )
 
+    def update(self, student, usage_key, course_key):
```

Comment:
No reason to take `student` as an input parameter since it was passed into the `__init__` method.